src/font/fdue.rs

```diff
@@ -71,7 +123,16 @@ impl FontBackend for Font {
                 ..Default::default()
             })
             .map(Font::from_fc_path)
-            .ok_or_else(|| anyhow::anyhow!("cannot find font"))?
+            .ok_or_else(|| {
+                let matching = Font::try_find_best_font(name);
+                println!("The font could not be found.");
```

Comment:
Maybe we should print this information using println too, because it's very uncomfortable and not intuitive use journalctl for finding the current problem with font name

Reply:
Added stdout forwarding in #75. stdout is not really available all the time, that's the problem